src/button-set.jsx

```diff
@@ -0,0 +1,57 @@
+// @flow
+import React, {PropTypes as t} from 'react';
+
+// Types
+type ButtonKindType =
+  | 'danger'
+  | 'default'
+  | 'info'
+  | 'primary'
+  | 'secondary'
+  | 'success';
+
+type ButtonType = {
+  text: string,
+  kind: ButtonKindType,
+  onClick: () => any
+};
+
+type ButtonSetPropType = {
```

Comment:
I think `Prop` should be `Props` because it is describing the type of the props object which is plural.